app/src/androidTest/java/xyz/jhughes/laundry/Matchers/RecyclerViewMatcher.java

```diff
@@ -0,0 +1,73 @@
+package xyz.jhughes.laundry.Matchers;
+
+import android.content.res.Resources;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+/**
+ * Espresso Recycler View Matcher
+ * https://github.com/dannyroa/espresso-samples/blob/master/RecyclerView/app/src/androidTest/java/com/dannyroa/espresso_samples/recyclerview/RecyclerViewMatcher.java
+ */
+
+public class RecyclerViewMatcher {
+    private final int recyclerViewId;
+
+    public RecyclerViewMatcher(int recyclerViewId) {
+        this.recyclerViewId = recyclerViewId;
+    }
+
+    public Matcher<View> atPosition(final int position) {
+        return atPositionOnView(position, -1);
+    }
+
+    public Matcher<View> atPositionOnView(final int position, final int targetViewId) {
```

Comment:
Is this trying to assert that a given item is at a given location? 

How does this work with the viewID, is that the view ID from the adapter or what?